### PR TITLE
Implement lint and release workflow via GitHub actions

### PR DESCRIPTION
**Pull request by Normo**
* move CI pipeline from GitLab CI/CD to GitHub Actions

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "Check out the codebase."
+        uses: "actions/checkout@v3"
+
+      - name: "Prepare the job environment."
+        uses: "./.github/workflows/prepare-action"
+
+      - name: "Lint code."
+        run: "poetry run ansible-lint -v --force-color --offline"
```
Comment (tobiashuste) [inline .github/workflows/ci.yml:38]: Is `yamllint` already run via `ansible-lint`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,7 +16,15 @@ Group your changes into these categories:
 
 `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `Security`.
 
-## [0.1.0](https://gitlab.com/hifis/ansible/rsd/-/releases/v0.1.0) - 2022-04-06
+## Unreleased
+
+[List of commits](https://github.com/hifis-net/ansible-role-rsd/compare/v0.1.0...main)
+
+* Implement lint and release workflow via GitHub actions
```
Comment (tobiashuste) [inline CHANGELOG.md:23]: Should we add a change category?

```suggestion
### Changed

* Implement lint and release workflow via GitHub actions
```

**Review by tobiashuste — CHANGES_REQUESTED** [review]
Thanks for implementing, I only added a question and a suggestion.

Is push mirroring already deactivated for the role?

File: CHANGELOG.md (hunk 2 above)
Comment (Normo) [reply CHANGELOG.md:23]: Of course!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Normo) [reply .github/workflows/ci.yml]: I explicitly added `yamllint`.

**Comment by Normo** [comment]
Thanks, @tobiashuste.

> Is push mirroring already deactivated for the role?
Yes, it is already disabled.

**Review by tobiashuste — APPROVED** [review]

**Comment by tobiashuste** [comment]
Thanks :)

**Pull request merged**